src/Tools/ExternalAccess/Razor/IRazorAsynchronousOperationListenerProviderAccessor.cs

```diff
@@ -4,12 +4,10 @@
 
 #nullable disable
 
-using Microsoft.CodeAnalysis.Shared.TestHooks;
-
 namespace Microsoft.CodeAnalysis.ExternalAccess.Razor
 {
     internal interface IRazorAsynchronousOperationListenerProviderAccessor
     {
-        IAsynchronousOperationListener GetListener(string featureName);
+        IRazorAsynchronousOperationListener GetListener(string featureName);
```

Comment:
💡 Use of an interface adds allocation overhead to this wrapper. Switching to a `readonly struct` will eliminate the overhead.